src/info/updates/archive.md

```diff
@@ -165,10 +176,21 @@ Our priorities are:
 ### Opportunities
 
 Do you have an idea that could transform and improve grants management and oversight? 10x, a federal idea accelerator, is [accepting proposals for new investment opportunities](https://10x.gsa.gov/posts/2023-submission-period/). All Federal employees are invited to submit ideas by November 30th.
-
-## Week of October 23, 2023
-
-The FAC participated in the Single Audit Roundtable, providing an update on our work to date and planned next steps, and used the opportunity to discuss pressing needs for auditors, resolution officials, and IGs.
+    </p>
```

Comment:
Another possibly-dangling paragraph tag. These are rendering as empty paragraph elements. Maybe that's intentional to clear the table? 

<img width="158" alt="image" src="https://github.com/GSA-TTS/FAC-transition-site/assets/2365503/fc1a881b-63cd-4f1f-9b61-0a2d3c2258e0">

```suggestion

```